### DISCUSSION TITLE
Windows 10 Search Box

### DISCUSSION THREAD

**Discussion by TimMarvelHero** [valinet/ExplorerPatcher#2973] [Q&A]
My friend downloaded Explorer Patcher and he doesn't have the search box option, I still have the search box and the option to change it in Explorer Patcher, has the update that added the search box been removed?

**Comment by haydengriz** [comment]
I still have it, but I’ve not updated, hopefully they haven’t since they said they were adding highlights to the search box

**Reply by TimMarvelHero** [reply]
yeah I haven't updated too
